.travis.yml

```diff
@@ -13,8 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-sudo: false
+sudo: required
 before_install: git fetch --unshallow
 language: java
+jdk:
+  - openjdk7
 install: mvn install --batch-mode -DskipTests=true -Dmaven.javadoc.skip=true -Dsource.skip=true > mvn_install.log || (cat mvn_install.log && false)
```

Comment:
@ilooner-mapr can you remove redirection to mvn_install.log as part of the same PR. It is not necessary.